cmd/podman/inspect.go

```diff
@@ -136,6 +136,7 @@ func iterateInput(ctx context.Context, size bool, args []string, runtime *adapte
 		case inspectTypeContainer:
 			ctr, err := runtime.LookupContainer(input)
 			if err != nil {
+				exitCode = 1
```

Comment:
Why are you using inspect to figure out if a container exists?
podman container exists CID
Does not work for you?

Reply:
In some cases, when we actually need to retrieve some attribute of a container, then it seems tempting to just rely on `podman inspect` and handle any failures as appropriate. We could separately invoke `podman container exists`. It would just be more typing, and we'd have the race condition where the container could get removed between the `container exists` and `inspect` calls by another concurrently running toolbox script or podman command.

Reply:
How are you presenting it to the user then?  Why can't you just take the output from 

podman inspect foobar
Error: error getting image "foobar": unable to find a name and tag match for foobar in repotags: no such image

What do you want your error message to look like?

Reply:
Sometimes it's not about throwing an error message about the container or image being absent. eg., when one does `toolbox list --containers`, we get a list of all the containers, and inspect the `State.Running` value for each of them so that we can colour running container entries in green. It's possible that we lose a race with something and the container has been deleted after we got the list and before we started iterating over it. In that case, the entry should just be skipped without printing an error.

Whatever the situation, we could detect this by looking at the standard error output of `podman inspect` and grepping for `no such image` or `no such container`. It just looks a bit fragile to screen scrape strings for error codes. A numeric exit code seems more idiomatic, but worst comes to worst, we can live with it.